src/sentry/conf/server.py

```diff
@@ -1601,6 +1601,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:issue-details-replay-event": False,
     # Enables syntax highlighting in the stack trace
     "organizations:issue-details-stacktrace-syntax-highlighting": False,
+    # Enables the new Stacktrace Link UI in frame header
+    "organizations:issue-details-stacktrace-link": False,
```

Comment:
Can you make a separate PR for the feature flag? Once that's merged we can test with prod data (I ask lazily since I don't want to set this up locally to try it out)

Also small nit, but I think we should name this a bit more specifically, like `issue-details-stacktrace-link-in-frame` so it's not confused with the existing feature